packages/webcomponents/README.md

```diff
@@ -0,0 +1,131 @@
+<p>
+  <br />
+  <a href="https://pangea.cloud?utm_source=github&utm_medium=node-sdk" target="_blank" rel="noopener noreferrer">
+    <img src="https://pangea-marketing.s3.us-west-2.amazonaws.com/pangea-color.svg" alt="Pangea Logo" height="40" />
+  </a>
+  <br />
+
+</p>
+
+<p>
+<br />
+
+[![documentation](https://img.shields.io/badge/documentation-pangea-blue?style=for-the-badge&labelColor=551B76)](https://pangea.cloud/docs/sdk/js/)
+[![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://pangea.cloud/join-slack/)
+
+<br />
+</p>
+
+## Motivation
+
+This library wraps both the [Audit Log Viewer](../react-mui-audit-log-viewer) and [Store File Viewer](../react-mui-store-file-viewer) react components in [webcomponents](https://www.webcomponents.org/introduction) and renders them using [preact](https://preactjs.com/) so that these components can be used outside of a [react](https://react.dev/) and [mui](https://mui.com/) context.
+
+## Getting Started
+
+This library exposes 2 registration functions (available globally if imported using `<script>`) that allow you to pass props to each component as if you were using the react components.
+Each prop declared by the react components is [bound](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) to the outer webcomponent that
+encapsulates them which allows a user to specify different themes, proxy functions, or other configuration options based on how they specify their attributes in their respective `.html` files.
+
+### Install
+
+You can install this by adding it to an `.html` file. You can copy paste the following in this directory and test using your browser:
```

Comment:
Either in addition to this, or replacing this, we'll need instruction on how to use this _outside_ of this directory. i.e. actually installing the package via npm.